rest-api/data/sites.csv

```diff
@@ -503,4 +576,787 @@ WALGREENS_MEMPHIS,hpo-site-walgreensmemphis10406,Walgreens Memphis Goodman Rd. W
 WALGREENS_MEMPHIS,hpo-site-walgreensmemphis10407,Walgreens Memphis Union Area,DV Walgreens Memphis 10407,7035735,,,,Active,Active,Active,Active,Active,Active,Active,Active,Active,Inactive,Active,Active,,,,Walgreens ,1863 Union Ave,,Memphis,TN,38104,1-855 Walgreens,,
 WALGREENS_MEMPHIS,hpo-site-walgreensmemphis10410,Walgreens Memphis Goodman Rd. Olive Branch,DV Walgreens Memphis 10410,7035735,,,,Active,Active,Active,Active,Active,Active,Active,Active,Active,Inactive,Active,Active,,,,Walgreens ,6958 Goodman Road,,Olive Branch,MS,38654,1-855 Walgreens,,
 WALGREENS_MEMPHIS,hpo-site-walgreensmemphis10411,Walgreens Memphis Winchester,DV Walgreens Memphis10411,7035735,,,,Active,Active,Active,Active,Active,Active,Active,Active,Active,Inactive,Active,Active,,,,Walgreens ,8001 Winchester Rd,,Memphis,TN,38125,1-855 Walgreens,,
-TEST_PROD,hpo-site-a,Prod HPO Site A,Prod HPO Site A,,,,,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,,,,,1515 N Campbell Ave,,Tucson ,AZ,85724,,,
\ No newline at end of file
+TEST_PROD,hpo-site-a,Prod HPO Site A,Prod HPO Site A,,,,,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,,,,,1515 N Campbell Ave,,Tucson ,AZ,85724,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,

```

Comment:
I don't think we want all these blank lines in here. Can you cut them and ask them to get rid of them in the source sheet?